### ISSUE TITLE
Masking Policy: The masking policy expression should not be "ForceNew"

### ISSUE THREAD

**Issue by blian7432** [chanzuckerberg/terraform-provider-snowflake#571]
**Provider Version**
0.25.5

**Terraform Version**
0.13.4

**Describe the bug**

In Masking Policy, the masking_expression should be updated without drop and re-create the resource. But, in pkg/resources/masking_policy.go, the maksing_expression schema is set to "ForceNew". So that changing a expression will cause drop masking policy. A masking policy can not be dropped once it is referenced by an object.  

**Expected behavior**

Remove "ForceNEW" in masking_expression schema. 

**Code samples and commands**
```
var maskingPolicySchema = map[string]*schema.Schema{
	"name": {
		Type:        schema.TypeString,
		Required:    true,
		Description: "Specifies the identifier for the masking policy; must be unique for the database and schema in which the masking policy is created.",
		ForceNew:    true,
	},
	"database": {
		Type:        schema.TypeString,
		Required:    true,
		Description: "The database in which to create the masking policy.",
		ForceNew:    true,
	},
	"schema": {
		Type:        schema.TypeString,
		Required:    true,
		Description: "The schema in which to create the masking policy.",
		ForceNew:    true,
	},
	"value_data_type": {
		Type:        schema.TypeString,
		Required:    true,
		Description: "Specifies the data type to mask.",
		ForceNew:    true,
	},
	"masking_expression": {
		Type:        schema.TypeString,
		Required:    true,
		Description: "Specifies the SQL expression that transforms the data.",
                ForceNew:    true, ######This should be removed
	},
	"return_data_type": {
		Type:        schema.TypeString,
		Required:    true,
		Description: "Specifies the data type to return.",
		ForceNew:    true,
	},
	"comment": {
		Type:        schema.TypeString,
		Optional:    true,
		Description: "Specifies a comment for the masking policy.",
	},
}
```
**Additional context**

**Comment by alldoami** [comment]
What errors do you get when changing the `masking_expression`? Does the object it is referencing drop the `masking_policy`?

**Comment by blian7432** [comment]
> What errors do you get when changing the `masking_expression`? Does the object it is referencing drop the `masking_policy`?

No. What is happening: 

1. I updated the masking policy expression 
2. The updated failed because Terraform try to drop the masking policy and re-create masking policy. 

What should happen: 
1. Masking policy is updated
2. Terraform use update script to modify the body. Not drop and re-create. 

Reason above happen: 
It puts "Force New" in masking expression schema. 

I have tested it in my local by remove "for new" in masking expression schema and it works.
